bucket/xmplay.json

```diff
@@ -0,0 +1,35 @@
+{
+    "version": "3.8.3.4",
+    "license": "Freeware",
+    "homepage": "https://www.un4seen.com/xmplay.html",
+    "description": "Win32 audio player with support for Winamp plugins.",
+    "url": "https://www.un4seen.com/files/xmplay38.zip",
+    "hash": "sha512:05ccbf67897b4ac55bc1591839ff559b55bd3c2db6aeaf3a4350a71c97485397a6a68fe5a54cf12cc2d51581b944444554d7adc3fe586fcba1195c6f2d4bacbe",
+    "bin": "xmplay.exe",
+    "shortcuts": [
+        [
+            "xmplay.exe",
+            "XMPlay"
+        ]
+    ],
+    "pre_install": [
+        "# Create placeholder files",
+        "function CreateFile([String[]] $file) {",
+        "    foreach ($f in $file) {",
+        "        if (! (Test-Path \"$dir\\$f\")) { New-Item \"$dir\\$f\" -ItemType 'File' | Out-Null }",
+        "    }",
+        "}",
+        "CreateFile 'xmplay.ini', 'xmplay.library', 'xmplay.pls'"
+    ],
+    "persist": [
+        "xmplay.ini",
+        "xmplay.library",
+        "xmplay.pls",
+        "user-plugins",
```

Comment:
Why manifest do not follow this structure?

![A](https://i.imgur.com/ns8LAsX.png)

http://support.xmplay.com/article.php?id=45

Reply:
Ok, it scans any folder for plugins and skins under same directory where it is.